extensions/amp-video/1.0/video-wrapper.js

```diff
@@ -216,22 +209,19 @@ function Autoplay({
           onClick={onOverlayClick}
         ></div>
       )}
-
-      {/* TODO(wg-bento): Global styling.
-          https://github.com/ampproject/wg-bento/issues/7 */}
-      <style>{autoplayCss}</style>
     </>
   );
 }
 
 /**
  * @return {!PreactDef.Renderable}
  */
-const AutoplayIconContent = once(() =>
-  [1, 2, 3, 4].map((i) => (
-    <div className="amp-video-eq-col" key={i}>
-      <div className={`amp-video-eq-filler amp-video-eq-${i}-1`}></div>
-      <div className={`amp-video-eq-filler amp-video-eq-${i}-2`}></div>
+const AutoplayIconContent = once(() => {
+  const classes = useAutoplayStyles();
+  return [1, 2, 3, 4].map((i) => (
+    <div className={`${classes.eqCol} amp-video-eq-col`} key={i}>
```

Comment:
whats the point of keeping `amp-video-eq-col`? Should probably just use `eqCol` instead right (can also update the `eqPlaying` rule above)?

Reply:
Technically this element's style is overridable via this class. Though I'm not sure any author has gone that deep 🤷‍♂️

Reply:
Ok cool! Just making sure theres a reason. Do you know if this will live in the shadow or light dom? If lightdom then it is worth keeping as-is, if shadow then I'm not sure they can customize it via classname anyway.

Reply:
It's using `['children']`, so it's rendered in ShadowDOM. Though its children are cloned into VNodes, so we could move it to the light DOM. In any case, I think it's fine to remove access to `amp-video-eq-col`, but maybe not to `amp-video-eq` since that may be used to hide the equalizer...